build-system/pr-check/checks.js

```diff
@@ -39,6 +39,12 @@ function pushBuildWorkflow() {
   timedExecOrDie('amp check-exact-versions');
   timedExecOrDie('amp check-renovate-config');
   timedExecOrDie('amp server-tests');
+  timedExecOrDie(
+    'amp make-extension --test --cleanup --name=amp-generated-for-test'
```

Comment:
optional nit: Since extensions are of the format `amp-foo` (`amp` + dash + contiguous extension name), is there value in using `--name=amp-foo` here instead of the longer name? (A nice cosmetic side effect might be that all this will fit on a single line.)

Reply:
Sounds good! Changed the argument to `for-test`, so each call is now on one line.

This gets used as `amp-for-test`, I changed the name flag's description to clarify.

Reply:
(Now using `t`. I think it's unlikely that we ever create `<amp-t>`.)